src/ServiceControl.UnitTests/BodyStorage/RavenAttachmentsBodyStorageTests.cs

```diff
@@ -23,6 +23,12 @@ public async Task Attachments_with_ids_that_contain_backslash_should_be_readable
 
                 var retrieved = await bodyStore.TryFetch(messageId);
                 Assert.True(retrieved.HasResult);
+                Assert.AreEqual(contentType, retrieved.ContentType);
```

Comment:
Should this be reset?

Reply:
Keeping - this makes the test more explicit, ensuring that this bug won't regress at the storage level.